spring-boot-autoconfigure-data-aerospike/src/main/java/org/springframework/boot/autoconfigure/data/aerospike/AerospikeDataConfiguration.java

```diff
@@ -98,46 +75,10 @@ public IndexRefresher aerospikeIndexRefresher(AerospikeClient aerospikeClient, I
         return refresher;
     }
 
-    @Bean(name = "mappingAerospikeConverter")
-    @ConditionalOnMissingBean(name = "mappingAerospikeConverter")
-    public MappingAerospikeConverter mappingAerospikeConverter(AerospikeMappingContext aerospikeMappingContext,
-                                                               AerospikeTypeAliasAccessor aerospikeTypeAliasAccessor,
-                                                               AerospikeCustomConversions aerospikeCustomConversions) {
-        return new MappingAerospikeConverter(aerospikeMappingContext, aerospikeCustomConversions, aerospikeTypeAliasAccessor);
-    }
-
-    @Bean(name = "aerospikeTypeAliasAccessor")
-    @ConditionalOnMissingBean(name = "aerospikeTypeAliasAccessor")
-    public AerospikeTypeAliasAccessor aerospikeTypeAliasAccessor(AerospikeDataProperties aerospikeDataProperties) {
-        return new AerospikeTypeAliasAccessor(aerospikeDataProperties.getTypeKey());
+    @Bean
+    @ConditionalOnMissingBean(name = "aerospikePersistenceEntityIndexCreator")
```

Comment:
have a weak experience of this project, but why the creation of these (and other) beans are not handled by configuration from spring-data project?

Reply:
spring-data-aerospike depends on spring only